@failfast
def addError(self, test, err):
    "Called when an error has occurred. 'err' is a tuple of values as\n        returned by sys.exc_info().\n        "
    self.errors.append((test, self._exc_info_to_string(err, test)))
    self._mirrorOutput = True
